packages/webview_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.4.5
+* Apply PlatformView API change.
+
 ## NEXT
 
 * Code refactoring.
```

Comment:
```suggestion
## 0.4.5

* Apply PlatformView API change.
* Code refactoring.
```

Reply:
Ok, I'll update this PR after https://github.com/flutter-tizen/plugins/pull/397 gets merged